lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -919,6 +924,7 @@ def setUp(self):
         self.request_factory = RequestFactory()
         self.user = UserFactory.create()
         self.request = self.request_factory.get("foo")
+        mako_middleware_process_request(self.request)
```

Comment:
Because the edxmako middleware leaves stuff in the cache (or the global module scope? I forget), which was leaking between tests. Run on their own, these tests don't pass without that change.